pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -70,6 +75,16 @@ type TFReplicaSpec struct {
 	RestartPolicy RestartPolicy `json:"restartPolicy,omitempty"`
 }
 
+// CleanPodPolicy describes how to deal with pods when the TFJob is finished.
```

Comment:
TFJob is finished  that include succeeded and failed ?

Reply:
Yeah, restarting is the internal state, thus succeeded and failed are all state to indicate that the TFJob is terminated.